glean-core/android/src/main/java/mozilla/telemetry/glean/config/Configuration.kt

```diff
@@ -110,10 +106,6 @@ data class Configuration internal constructor(
          * The default server pings are sent to.
          */
         const val DEFAULT_TELEMETRY_ENDPOINT = "https://incoming.telemetry.mozilla.org"
-        /**
-         * The default user agent used when sending pings.
-         */
-        const val DEFAULT_USER_AGENT = "Glean/${BuildConfig.LIBRARY_VERSION} (Android)"
```

Comment:
Can you double check that the `BuildConfig.LIBRARY_VERSION` value is the same we'd see in glean-core? It should be :)

Reply:
Yes it is.